rps.js

```diff
@@ -0,0 +1,45 @@
+function playFunction() {
+    const userChoice = prompt("Do you choose rock, paper or scissors?");
+    const computerChoice = Math.random();
+    if (computerChoice < 0.34) {
+        computerChoice = "rock";
```

Comment:
The value of a constant can't be changed through reassignment, and it can't be redeclared.